### DISCUSSION TITLE
Multiple live iso-Images to choose from

### DISCUSSION THREAD

**Discussion by feschoppe** [ltsp/ltsp#255] [Q&A]
Hello everyone,
is there a way to deliver multiple live systems to netboot without ltsp mode?
I followed these steps to netboot one live iso:
https://github.com/ltsp/community/wiki/Non-LTSP-iPXE-entries
It works pretty fine, but I would like to netbook several different isos. What’s the best way to do that?

**Comment by alkisg** [comment]
Maybe https://netboot.xyz/ ?

I mean, you could surely do what I wrote in that example wiki page multiple times, but you'd need to find the correct command lines for all distributions / versions etc (one line per distro), while netboot.xyz already has those so it will save you some effort.
